README.md

```diff
@@ -97,11 +97,16 @@ Log in as the buildbot user, and do the following:
     virtualenv --no-site-packages buildbot-env
     source buildbot-env/bin/activate
     echo "export PATH=/home/buildbot/buildbot-ros/scripts:${PATH}" >> buildbot-env/bin/activate
-    easy_install buildbot
+    easy_install buildbot=0.8.12 requests
     pip install rosdistro
     git clone git@github.com:mikeferguson/buildbot-ros.git
     buildbot create-master buildbot-ros
 
+If using the Pull Request builder, you will also need to:
+
+    sudo apt-get install libffi-dev libssl-dev
+    easy_install txgithub
```

Comment:
I remember having to install a few extra packages and do some tweaking, but I don't think I wrote anything elaborate down.

I also did a bunch of fixes in troubleshooting attempts, so the best bet is either to change the docs after updating and seeing what comes up or for me installing it on one of our unused computers and adding as I go along.

> On Jul 11, 2015, at 4:42 PM, Michael Ferguson <notifications@github.com> wrote:
> 
> In README.md:
> 
> >      pip install rosdistro
> >      git clone git@github.com:mikeferguson/buildbot-ros.git
> >      buildbot create-master buildbot-ros
> >  
> > +If using the Pull Request builder, you will also need to:
> > +
> > +    sudo apt-get install libffi-dev libssl-dev
> > +    easy_install txgithub
> @mehwang are there any other things that need to be installed? (this is based on my current test run)
> 
> —
> Reply to this email directly or view it on GitHub.
>